### PR TITLE
feat: delete default port options that has defined in egg

### PR DESCRIPTION
**Pull request by popomore**
##### Checklist

- [x] `npm test` passes
- [x] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [x] commit message follows commit guidelines

##### Description of change

PR #34 has implemented setting default port in config,
so egg-cluster should not set the default port.

Closes https://github.com/eggjs/egg/issues/1371

### CODE REVIEW

File: lib/utils/options.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ module.exports = function(options) {
   const defaults = {
     framework: '',
     baseDir: process.cwd(),
-    port: options.https ? 8443 : 7001,
+    port: options.https ? 8443 : null,
```
Comment (atian25) [inline lib/utils/options.js]: 如果外面没传，listenConfig.port 有默认值么？

Comment (popomore) [reply lib/utils/options.js]: egg 有

Comment (atian25) [reply lib/utils/options.js]: ok

**Comment by popomore** [comment]
egg-cluster 使用 egg-mock 测试无法覆盖这个问题，感觉应该去掉 egg-mock

**Comment by atian25** [comment]
@popomore 合并了?

> egg-cluster 使用 egg-mock 测试无法覆盖这个问题，感觉应该去掉 egg-mock

这个还要改啥?

**Comment by popomore** [comment]
+ egg-cluster@1.11.0

**Pull request merged**